hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -917,13 +917,13 @@ possible configurations would overwhelm and obscure the important.
   <property>
     <name>hbase.offpeak.start.hour</name>
     <value>-1</value>
-    <description>The start of off-peak hours, expressed as an integer between 0 and 23, inclusive.
+    <description>The start of off-peak hours, expressed as an integer between 0 and 24, inclusive.
       Set to -1 to disable off-peak.</description>
   </property>
   <property>
     <name>hbase.offpeak.end.hour</name>
     <value>-1</value>
-    <description>The end of off-peak hours, expressed as an integer between 0 and 23, inclusive. Set
+    <description>The end of off-peak hours, expressed as an integer between 0 and 24, exclusive. Set
```

Comment:
How is this `exclusive` for 24 based on `return 0 <= hour && hour <= 24; ` condition?

Reply:
`    @Override
    public boolean isOffPeakHour(int targetHour) {
      if (startHour <= endHour) {
        return startHour <= targetHour && targetHour < endHour;
      }
      return targetHour < endHour || startHour <= targetHour;
    }`
See the code, it will be "return 0 <= hour && hour < 24"